csharp/.editorconfig

```diff
@@ -0,0 +1,29 @@
+# top-most EditorConfig file
+root = true
+
+# All Files
+[*]
+insert_final_newline = true
+trim_trailing_whitespace = true
+
+# C# Files
+[*.{csproj}]
+indent_size = 2
+indent_style = space
+
+# C# Files
+[*.{cs}]
+indent_size = 4
+indent_style = tab
```

Comment:
Can you switch C# indentation style to space? See [here](https://learn.microsoft.com/en-us/dotnet/csharp/fundamentals/coding-style/coding-conventions) for conventions.

Reply:
Updated to space, thank you for checking.